fixtures/README.md

```diff
@@ -0,0 +1,40 @@
+# Fixtures Files
+
+## Purpose
+
+We have collected various JS files from a variety of public open source projects on GitHub.
+
+These files help us ensure that future updates to this ESLint config/plugin do not break existing code that passes linting. When a new rule is created, or when an existing rule is updated, these files will help make us aware of scenarios where the updates do not work as expected.
+
+## What Happens If a Plugin/Config Update Breaks Fixtures?
+
+- If the rule is catching something that is not actually a problem:
+  - Reconfigure, disable, or modify the lint rule
+- If the rule is catching something that is actually a problem in the fixtures code:
+  - Whichever is easiest:
+    - [Add `// eslint-disable-next-line some-rule` comments](https://eslint.org/docs/2.13.1/user-guide/configuring#disabling-rules-with-inline-comments)
+    - Fix the error manually
+    - Delete the code with the lint error. The fixture files don't have to _work_, so sometimes this is the easiest option.
+
+## Finding Projects to Use as Fixtures
+
+- Try to use repos that come from a diverse set of authors or projects rather than a bunch of related projects from the same author
+- Try to use repos from different parts of the javascript ecosystem (browser, node, transpiling, etc.)
+- Try to use repos that use different parts and versions of javascript syntax
+
+[`awesome-nodejs`](https://github.com/sindresorhus/awesome-nodejs) and [`awesome-javascript`](https://github.com/sorrycc/awesome-javascript) have lists of packages that could be used.
+
+## Adding Fixture Files
+
+1. From the project root, run `./fixtures/load-repo.js`
```

Comment:
How do I run this? I assume `node ./fixtures/load-repo.js` but it'd be nice if the readme was clear about this

Reply:
You don't need `node`. Just `./fixtures/load-repo.js` should work (the file is marked as executable)

Reply:
I got confused by this as well. 😅

Reply:
Added an npm script to reduce confusion 🎉